tests/transformers/test_transformers_model_export.py

```diff
@@ -1109,6 +1110,7 @@ def test_transformers_tf_model_save_without_conda_env_uses_default_env_with_expe
     pip_requirements = _get_deps_from_requirement_file(model_path)
     assert "tensorflow" in pip_requirements
     assert "torch" not in pip_requirements
+    assert "accelerate" not in pip_requirements
```

Comment:
```suggestion
    assert "accelerate" in pip_requirements
```

Is this line correct?

Reply:
Ah I see, this is a test for TF.